File: documentation/user/en/use/api/example/update-existing-entity-shortened.java
session.getEntity("Product", 1, attributeContentAll(), dataInLocales(Locale.ENGLISH))
	.orElseThrow(
		() -> new IllegalArgumentException("Product `1` not found!")
	)
	.openForWrite()
	.setAttribute(
		"name", Locale.ENGLISH,
		"ASUS Vivobook 16 X1605EA-MB044W Indie Black"
	)
	.upsertVia(session);

File: documentation/user/en/use/api/example/update-existing-entity-shortened.cs
session.GetEntity("Product", 1, AttributeContentAll(), DataInLocales(new CultureInfo("en")))
	.OpenForWrite()
	.SetAttribute(
		"name", new CultureInfo("en"),
		"ASUS Vivobook 16 X1605EA-MB044W Indie Black"
	)
	.UpsertVia(session);